docs/source/user_guide/runtime-image-conf.md

```diff
@@ -0,0 +1,125 @@
+
+Runtime image configurations can be added, modified, and removed in the _Runtime Images_ panel.
+
+To access the panel in JupyterLab:
+- Open the JupyterLab command palette (`<cmd/ctrl><shift><c>`).
+- Click `Show Runtime Images` in the `Elyra` section.
+
+![Runtime Images UI](../images/runtime-images-ui.png)
+
+To add a runtime image configuration:
+- Click `+` to add a runtime image.
+- Add the runtime image properties as appropriate.
+
+To edit a runtime image configuration:
+- Click the `edit` icon next to the runtime image name.
+- Modify the runtime image properties as desired.
+
+To delete a runtime image configuration:
+- Click the `delete` icon next to the runtime image name.
+- Confirm deletion.
+
+#### Managing Runtime Images with the Command Line Interface
+
+Runtime image configurations can be added, replaced, and removed with the `elyra-metadata` command line interface.
+
+To list runtime image configurations:
+
+```bash
+$ elyra-metadata list runtime-images
+
+Available metadata instances for runtime-images (includes invalid):
+
+Schema          Instance               Resource                                                                                                       
+------          --------               --------                                                                                                       
+runtime-image   anaconda               /Users/jdoe/.../jupyter/metadata/runtime-images/anaconda.json
+...  
+```
+
+To add a runtime image configuration for the public `jdoe/my-image` Docker image:
+
+```bash
+$ elyra-metadata install runtime-images --schema_name=runtime-image \
+       --name="my_image_name" \
+       --display_name="My runtime image" \
+       --description="My custom runtime Docker image" \
+       --image_name="jdoe/my-image"
+```
+
+To replace a runtime image configuration append the `--replace` option:
+
+```bash
+$ elyra-metadata install runtime-images --schema_name=runtime-image \
+       --name="my_image_name" \
+       --display_name="My runtime image" \
+       --description="My other custom runtime Docker image" \
+       --image_name="jdoe/my-other-image" \
+       --replace
+```
+
+To delete a runtime image configuration:
+
+```bash
+$ elyra-metadata remove runtime-images \
+       --name="my_image_name"
+```
+
+
+#### Configuration properties 
+
+The runtime image configuration properties are defined as follows. The string in the headings below, which is enclosed in parentheses, denotes the CLI option name.
+
+##### Name (display_name)
+A user-friendly name for runtime image definition. This property is required.
+
+Example: `My runtime image`
+
+##### Description (description)
+Desscription for this runtime image definition.
+
+Example: `My custom runtime Docker image`
+
+##### Image Name (image_name)
+The name of an existing Docker image in the public Docker Hub container registry that meets the stated prerequisites. This property is required.
+
+Example: `jdoe/my-image`
```

Comment:
I would recommend using the version tag to make things more reproducible, otherwise, you might start getting new "latest" versions when those become available.

Reply:
ah yes, of course. excellent point! Will change the text and the screen capture.